README.md

```diff
@@ -91,11 +84,13 @@ Every voice is important. If you have something on your mind, create an issue or
 
 ## Tested with Ansible
 
-Ansible-core 2.15, 2.16, 2.17, 2.18, and the current development version of ansible-core.
+Ansible-core 2.17, 2.18, and the current development version of ansible-core.
 
 ## External requirements
+In order to use the modules in this collection, you'll need:
 
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
+  * Python >= 3.7
+  * Proxmoxer >= 2.0
```

Comment:
Can you also adjust `requires_ansible` in `meta/runtime.yml`? That's the main location used by ansible-core, Galaxy, and antsibull-nox to figure out what's supported :)